default-config-inputs.json

```diff
@@ -10,9 +10,10 @@
     "settings": {
         "postureControlInputs": {
             "imageRepositoryAllowList": [
-                "ecr.*amazonaws.com",
+                "amazonaws.com",
                 ".*.gcr.io",
-                ".*azurecr.io"
+                "azurecr.io",
+                "docker.io"
```

Comment:
@slashben @Daniel-GrunbergerCA Is there a reason docker was not on the list to begin with?

Reply:
Not that I'm aware of - but docker.io for implicit docker hub image case was not handled anyway before this update.
In any case, I understood from @slashben that we are aiming to eventually leave this field with empty list and let the user to explicitly configure it.